lib/core/src/client/preview/loadCsf.test.ts

```diff
@@ -246,13 +254,19 @@ describe('core.preview.loadCsf', () => {
 
     const mockedStoriesOf = clientApi.storiesOf as jest.Mock;
     const aApi = mockedStoriesOf.mock.results[0].value;
-    expect(aApi.add).toHaveBeenCalledWith('X', input.a.x, {
-      x: 'y',
-      decorators: [decorator],
-      __id: 'a--x',
-      args: { b: 1 },
-      argTypes: { b: 'string' },
-    });
+    expect(aApi.add).toHaveBeenCalledWith(
+      'X',
+      input.a.x,
+      {
+        x: 'y',
+        decorators: [decorator],
+        __id: 'a--x',
+        args: { b: 1 },
+        argTypes: { b: 'string' },
+      },
+      undefined,
```

Comment:
This doesn't seem right to me, seems like tags should be part of the 3rd argument (the object)